src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreInstrumentationOptions.cs

```diff
@@ -34,15 +34,31 @@ public class AspNetCoreInstrumentationOptions
         public Func<HttpContext, bool> Filter { get; set; }
 
         /// <summary>
-        /// Gets or sets an action to enrich an Activity.
+        /// Gets or sets an action to enrich an Activity during start event.
```

Comment:
I'd avoid the "during start event", unless we clearly state what is "start". Probably best to stick with existing summary.